src/makeCalculator.js

```diff
@@ -38,6 +38,41 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, number) {
+      this.result = callback(number, this.result);
+
+      return this;
+    },
+
+    add(number1, number2) {
+      return number2 + number1;
+    },
+
+    subtract(number1, number2) {
+      return number2 - number1;
+    },
+
+    multiply(number1, number2) {
+      return number2 * number1;
+    },
+
+    divide(number1, number2) {
+      if (number2 !== 0) {
```

Comment:
you should check the **first number** for 0 as its a denominator and it can't be 0